crates/json-sql/SmartModule.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "json-sql"
 group = "infinyon"
-version = "0.1.0"
+version = "0.1.1"
```

Comment:
Not critical, but I would bump a minor version. New big functionality added

Reply:
yeah makes sense, done